packages/bdd-tests/util/helper.ts

```diff
@@ -191,14 +236,14 @@ export async function sendDispenseNotification(site, medDispNo = 1, table){
 export async function amendDispenseNotification(itemNo, table){
   data.id = crypto.randomUUID();
   data.identifier.value = crypto.randomUUID();
-  let ext = misc.extReplacementOf
-  ext.extension[0].valueIdentifier.value = identifierValue
+  const ext = misc.extReplacementOf
+  console.log("our test + " + ext.extension[0].valueIdentifier.value )
```

Comment:
Does this line need reverting back to:
ext.extension[0].valueIdentifier.value = identifierValue ?